### PR TITLE
KEYCLOAK-13246 Update Italian translations

### PR DESCRIPTION
**Pull request by axl8713**

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/account/messages/messages_it.properties [hunk 1]
```diff
@@ -71,83 +103,241 @@ date=Data
...
+authenticatorSMSMessage=Keycloak invier\u00E0 il codice di verifica al tuo telefono per l''autenticazione a due fattori.
+authenticatorSMSFinishSetUpMessage=I messaggi di testo vengono inviati a
+authenticatorDefaultStatus=Default
+authenticatorChangePhone=Cambia numero di telefono
+authenticatorBackupCodesTitle=Codici di backup
+authenticatorBackupCodesMessage=Ottieni i tuoi codici di backup a otto cifre
+authenticatorBackupCodesFinishSetUpMessage=Sono stati generati dodici codici di backup. Ognuno pu\u00f2 essere usato una sola volta.
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=Setup autenticatore mobile
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_it.properties:288]: Setup autenticatore per telefono cellulare

File: themes/src/main/resources-community/theme/base/account/messages/messages_it.properties [hunk 2]
```diff
@@ -71,83 +103,241 @@ date=Data
...
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=Setup codice SMS
+chooseYourCountry=Scegli la tua nazione
+enterYourPhoneNumber=Inserisci il tuo numero di telefono
+sendVerficationCode=Invia il codice di verifica
+enterYourVerficationCode=Inserisci il codice di verifica
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=Setup backup codici
+backupcodesIntroMessage=Se non disponi pi\u00f9 del tuo telefono, puoi comunque accedere al tuo account attraverso i codici di backup. Conservai in un posto sicuro e accessibile.
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_it.properties:307]: Se non disponi pi\u00f9 del tuo telefono, puoi comunque accedere al tuo account attraverso i codici di backup. Conservali in un posto sicuro e accessibile.

File: themes/src/main/resources-community/theme/base/account/messages/messages_it.properties [hunk 3]
```diff
@@ -71,83 +103,241 @@ date=Data
...
+realmName=Realm
+doDownload=Download
+doPrint=Stampa
+backupCodesTips-1=Ogni codice di backup pu\u00f2 essere usato una sola volta.
+backupCodesTips-2=Questi codici sono stati generati il
+generateNewBackupCodes=Genera dei nuovi codici di backup
+backupCodesTips-3=Quando generi dei nuovi codici di backup, quelli attuali non funzionano pi\u00f9.
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_it.properties:314]: Quando generi dei nuovi codici di backup, quelli attuali non funzioneranno pi\u00f9.

File: themes/src/main/resources-community/theme/base/account/messages/messages_it.properties [hunk 4]
```diff
@@ -71,83 +103,241 @@ date=Data
...
+perPage=per pagina
+currentPage=Pagina corrente
+sharetheResource=Condividi la risorsa
+group=Gruppo
+selectPermission=Seleziona permessi
+addPeople=Aggiungi persone con le quali condividere la tua risorsa
+addTeam=Aggiungi gruppi con i quali condividere la tua risorsa
+myPermissions=Miei permessi
+waitingforApproval=Attesa dell''approvazione
+anyPermission=Qaualsiasi permesso
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_it.properties:337]: Qualsiasi permesso

File: themes/src/main/resources-community/theme/base/account/messages/messages_it.properties [hunk 5]
```diff
@@ -71,83 +103,241 @@ date=Data
...
+additionalGrants=Ulteriori concessioni
 action=Azione
 inResource=in
 fullAccess=Accesso completo
 offlineToken=Token offline
 revoke=Revoca concessione
 
-configureAuthenticators=Configura Authenticators
+configureAuthenticators=Autenticatori configurati
 mobile=Mobile
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_it.properties:132]: Mobile is better translated with "dispositive mobile" or "telefono cellulare"

File: themes/src/main/resources-community/theme/base/login/messages/messages_it.properties [hunk 6]
```diff
@@ -75,17 +116,23 @@ emailVerifyInstruction1=Ti \u00e8 stata inviata una email con le istruzioni per
...
+emailLinkIdp5=per continuare.
 
 backToLogin=&laquo; Torna al Login
 
 emailInstruction=Inserisci la tua username o l''indirizzo email e ti manderemo le istruzioni per creare una nuova password.
 
 copyCodeInstruction=Copia questo codice e incollalo nella tua applicazione:
 
+pageExpiredTitle=La pagina \u00e8 scaduta
+pageExpiredMsg1=Per riavviare il login
```
Comment (axgus) [inline themes/src/main/resources-community/theme/base/login/messages/messages_it.properties:133]: Per rifare il login

**Comment by stianst** [comment]
@axgus  thanks for reviewing this. @axl8713 could you address the comments please?

**Comment by axl8713** [comment]
> @axgus thanks for reviewing this. @axl8713 could you address the comments please?

@stianst sure. I'll be doing this shortly.

**Comment by stianst** [comment]
@axl8713 thanks, and sorry for the long time it's taken to get around to picking this PR up

File: themes/src/main/resources-community/theme/base/login/messages/messages_it.properties (hunk 6 above)
Comment (axl8713) [reply themes/src/main/resources-community/theme/base/login/messages/messages_it.properties:133]: What about this?
```suggestion
pageExpiredMsg1=Per ripetere il login
```

**Comment by stianst** [comment]
Looks like there was changes here since the last review. @axl8713 could you confirm if you have addressed the issues? @axgus could you review again please?

**Comment by axl8713** [comment]
@stianst yes. I've made corrections to other problems found while addressing the issues.

**Review by stianst — APPROVED** [review]

**Pull request merged**